src/pages/user-experience-design/a-b-testing/index.md

```diff
@@ -1,15 +1,14 @@
 ---
 title: A B Testing
 ---
-## A B Testing
+## What is A B Testing?
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/user-experience-design/a-b-testing/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+A/B testing (sometimes called split testing) is comparing two versions of a web page or element to see which one performs better. You compare two web pages or elements by showing the two variants (let's call them A and B) to similar visitors at the same time. The one that gives a better conversion rate, wins!
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
-
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+Every website wants visitors converting from just visitors to something else(Convertions, sign ups, or just take action for something). The rate at which a website is able to do this is its "conversion rate". Measuring the performance of a variation (A or B) means measuring the rate at which it converts visitors to goal achievers.
 
 #### More Information:
 <!-- Please add any articles you think might be helpful to read before writing the article -->
-
+For more information on A/B Testing please read
+<a href='https://vwo.com/ab-testing/' target='_blank'>The Complete Guide to A/B Testing</a>.
```

Comment:
This link currently doesn't work anymore (or at least right at this moment when I tried going to it).